aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -2592,62 +2564,58 @@ module aptos_framework::stake {
         leave_validator_set(validator_2, validator_2_address);
         join_validator_set(validator_3, validator_3_address);
         // Validator 2 is not effectively removed until next epoch.
-        assert!(get_validator_state(validator_2_address) == VALIDATOR_STATUS_PENDING_INACTIVE, 6);
+        assert!(get_validator_state(validator_2_address)
+            == VALIDATOR_STATUS_PENDING_INACTIVE, 6);
         assert!(
-            vector::borrow(
-                &borrow_global<ValidatorSet>(@aptos_framework).pending_inactive,
-                0
-            ).addr == validator_2_address,
-            0
-        );
+            vector::borrow(&borrow_global<ValidatorSet>(@aptos_framework).pending_inactive,
+                0).addr
+            == validator_2_address,
+            0);
         // Validator 3 is not effectively added until next epoch.
-        assert!(get_validator_state(validator_3_address) == VALIDATOR_STATUS_PENDING_ACTIVE, 7);
+        assert!(get_validator_state(validator_3_address)
+            == VALIDATOR_STATUS_PENDING_ACTIVE, 7);
         assert!(
-            vector::borrow(
-                &borrow_global<ValidatorSet>(@aptos_framework).pending_active,
-                0
-            ).addr == validator_3_address,
-            0
-        );
+            vector::borrow(&borrow_global<ValidatorSet>(@aptos_framework).pending_active,
+                0).addr
```

Comment:
Putting `).addr` into a newline seems better?

Reply:
Your idea aligns with human intuition, but there are two issues:


1. Dot access occurs in many scenarios, and it is technically difficult to specifically identify situations that can be treated specially.
2. Similar to rustfmt, if each element within a parenthesis expression occupies its own line, it is necessary to move the right parenthesis to a new line. However, this scenario conflicts with this rule in our movefmt implementation. Simply put, it might be the gap between ideal and reality, as is often the case in the world.